gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/AndroidCommandPluginExtension.groovy

```diff
@@ -77,6 +78,10 @@ public class AndroidCommandPluginExtension {
         System.properties['seed'] ?: seed
     }
 
+    def getCategories() {
+        System.properties['categories'] ?: categories ?: ''
```

Comment:
lol not ternary tho. Elvis operator. Still a bit strange but not nearly so objectionable as a nested ternary.